less/tree.less

```diff
@@ -1,272 +1,271 @@
 @import "./Common/Constants";
 
-
 .resourceTree {
+  height: 100%;
+  flex: 0 0 auto;
+  width: 20%;

```

Comment:
Verify that the collapsing the resource tree still works. I know this is something we broke earlier. The tabs wouldn't fill the space.

Reply:
This will indeed break collapsing the resource tree. I've reverted the change.